src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
I can totally do that! I wasn't sure about babel on the React codebase but as @gaearon pointed out, babel is there so, yes I can do it. (After this, I can also start to add some flow throughout the codebase if you are ok with that, you can even point me to critical parts. However, the flow declarations for react already are super complete, so I'm not sure there is really critical parts).